Library/Photo.swift

```diff
@@ -1,7 +1,7 @@
 import UIKit
 import Photos
 
-struct Section {
+class Section {
```

Comment:
[Here](https://github.com/bakkenbaeck/Viewer/blob/41f1eba26b8ed7cee6d21deb09e98e6bfda6c02c/Library/PhotosController.swift#L159-L166) I'm modifying an element from the array, if it's a struct the original element doesn't get modified, just a copy of it, so you have to insert it back into the array and remove the old copy. This was easily avoided using `class`, then any modification to an object results in the actual object being modified.

<img width="660" alt="screen shot 2017-04-19 at 11 37 25" src="https://cloud.githubusercontent.com/assets/1088217/25173498/9b9bab52-24f4-11e7-81b4-4f28d8a111a4.png">